draft-jholland-quic-multicast.md

```diff
@@ -324,7 +325,7 @@ MC_CHANNEL_PROPERTIES frames contain the following fields:
 
   * ID Length: The length in bytes of the Channel ID field.
   * Channel ID: The channel ID for the channel associated with this frame.
-  * Properties Sequence Number: Increases by 1 each time the properties for the channel are changed by the server.  The client tracks the sequence number of the MC_CHANNEL_PROPERTIES frame that set its current value, and only updates the value and the packet number range on which it's applicable if the Properties Sequence Number is higher.
+  * Properties Sequence Number: Increases by 1 each time the properties for the channel are changed by the server.  The client tracks the sequence number of the MC_CHANNEL_PROPERTIES frame that set it is current value, and only updates the value and the packet number range on which it's applicable if the Properties Sequence Number is higher.
```

Comment:
This is a wrong edit.  "its" could be changed to "the channel's" if its meaning is unclear, but this was a possessive pronoun, not a conjunction of "it is".